docs/specs/moniker.yml

```diff
@@ -838,6 +838,37 @@ outputs:
       "metadata": {"monikers": ["netcore-1.1", "netcore-1.2", "netcore-1.3" ]}
     }
 ---
+# Build toc with monikers info - multiple node with same url
+inputs:
+  docfx.yml: |
+    monikerRange:
+      'docs/v1/**': 'netcore-1.0'
+      'docs/v2/**': 'netcore-2.0'
+      'docs/v3/**': 'netcore-1.0 || netcore-2.0'
```

Comment:
For readability
```suggestion
      'docs/v1-v2/**': 'netcore-1.0 || netcore-2.0'
```

Reply:
OK, fix in next PR

---
In reply to: [293207140](https://github.com/dotnet/docfx/pull/4702#discussion_r293207140) [](ancestors = 293207140)